### ISSUE TITLE
4.0.0 Beta 2 Windows installer: Blocked by SmartScreen (Windows 10)

### ISSUE THREAD

**Issue by genodeftest** [exaile/exaile#401]
Windows puts a scary warning which might prevent users from installing Exaile.

### Steps to Reproduce (for bugs)
1. Have Windows 10 (might need to be fully updated)
2. Download Exaile 4.0.0 Beta 2
3. Start the installer

### Expected Behavior
Install with just the usual UAC confirmation dialog, but without a scary warning.

### Current Behavior
Windows dialog (rough translation from German to English):

> This computer is being protected by Windows
> 
> Windows SmartScreen has prevented an unknown app from being executed. Executing this app poses a potential thread to this PC.
> App: exaile-4.0.0-beta2.exe
> Vendor: Unknown vendor

After clicking on the "more information" button, there is a "run anyway" button, which works fine. The installer is not technically broken.

### Possible Solution
Sign the installer or get it signed by somebody. This may or may not help.

### Environment
Windows 10 Pro N, with most or all updates installed as of 2017-06-16.

**Comment by virtuald** [comment]
Maybe we can set the 'vendor' metadata in the manifest somewhere, and that would quiet SmartScreen.

**Comment by sjohannes** [comment]
I don't think there's much we can do about SmartScreen, aside from paying for a code signing certificate and signing our builds. And even then, unless the cert is an EV cert (more expensive, more hassle) it still needs to build reputation. [More info at the IE team blog.](https://blogs.msdn.microsoft.com/ie/2012/08/14/microsoft-smartscreen-extended-validation-ev-code-signing-certificates/)

Wontfix?

**Comment by genodeftest** [comment]
Why did it work fine then with the Beta 1?

**Comment by sjohannes** [comment]
I don't know; perhaps enough people have downloaded/run it, so that particular file is now "trusted". At least that's how I think the reputation is supposed to work for unsigned binaries.

**Comment by genodeftest** [comment]
> I don't know; perhaps enough people have downloaded/run it, so that particular file is now "trusted". At least that's how I think the reputation is supposed to work for unsigned binaries.

Seems legit. If that's the case, this is a minor issue.

**Comment by virtuald** [comment]
I think we should ignore this for now unless there's a significant issue in the future.
